react-app/src/components/PortfolioScreen/PortfolioPanel.tsx

```diff
@@ -69,7 +69,7 @@ const CoinsAmountField: React.FC<{
   amount: BigNumber;
 }> = ({ messageID, amount }) => {
   return (
-    <div className={cn("flex", "flex-col", "mr-6")}>
+    <div className="flex flex-col sm:mb-4 sm:w-1/3 lg:mr-6 lg:w-auto">
```

Comment:
where did the `w-1/3` and `mb-4` come from

also there should only be one breakpoint - `sm`, shouldn't need special handling for an extra breakpoint

Reply:
as discussed offline, will use grid instead